### DISCUSSION TITLE
Next Step to WAYLAND

### DISCUSSION THREAD

**Discussion by zakariaGatter** [herbstluftwm/herbstluftwm#1572] [General]
Hello Everyone 

as we see the rapid transition from xorg to wayland, is there any future plans for wayland i love herbstluftwm and it will be nice to have it on wayland too?

NOTE: just a question not a command or order

**Comment by t-wissmann** [comment]
Hi,
I'd definitely like to extend hlwm to wayland, but I don't have the time to do it beside my job and family right now. I work at a university and I'm involved in teaching, so currently, I invest all my free time in research. Maybe, things relax in the future, but I can't make any promises. For more comments on the technical side, see [this comment in issue tracker](https://github.com/herbstluftwm/herbstluftwm/issues/1569#issuecomment-1826021267) (Thanks for your kind 'note' by the way :-) ).
